resolwe/flow/static/flow/processorSchema.json

```diff
@@ -114,6 +114,32 @@
                         }
                     },
                     "additionalProperties": false
+                },
+                "resources": {
+                    "description": "Resource requirements for this process",
+                    "type": "object",
+                    "properties": {
+                        "cores": {
+                            "description": "Number of CPU cores",
+                            "type": "integer"
+                        },
+                        "memory": {
+                            "description": "Amount of memory in megabytes",
+                            "type": "integer"
+                        },
+                        "temporary_storage": {
```

Comment:
Do you plan to tackle these two (`temporary_storage` and `output_storage`) in this PR or later?